src/Base/Quantity.cpp

```diff
@@ -299,7 +299,7 @@ Quantity Quantity::Ounce            (0.0283495231  ,Unit(0,1));
 Quantity Quantity::Stone            (6.35029318    ,Unit(0,1));
 Quantity Quantity::Hundredweights   (50.80234544   ,Unit(0,1));
 
-Quantity Quantity::PoundForce       (224.81        ,Unit(1,1,-2)); // Newton  are ~= 0.22481 lbF
+Quantity Quantity::PoundForce       (4448.22       ,Unit(1,1,-2)); // lbF are ~= 4.44822 Newton
```

Comment:
While you're at it, maybe you could change `lbF` to `lbf`, [to make it match with the unit you write](https://github.com/FreeCAD/FreeCAD/blob/master/src/App/ExpressionParser.l#L244).
```suggestion
Quantity Quantity::PoundForce       (4448.22       ,Unit(1,1,-2)); // lbf are ~= 4.44822 Newton
```